order.go

```diff
@@ -69,12 +69,14 @@ type Order struct {
 	OrderNumber           int              `json:"order_number"`
 	Note                  string           `json:"note"`
 	Test                  bool             `json:"test"`
+	Gateway               string           `json:"gateway"`
```

Comment:
As I mentioned elsewhere, this field is deprecated since 2014, so I don't think we should add to avoid someone becoming reliant on it.